### PR TITLE
fixed order typo, added user validations

### PR DESCRIPTION
**Pull request by nataliarossini**
hey guys I've created a few validations although I'm not sure about user has many cakes or user has many cakes through orders, we can discuss it tomorrow, I also create some security validations for the password

### CODE REVIEW

**Comment by nataliarossini** [comment]
Oh also changed only a ':' that had a extra space in the orders model validation and it was getting a rubocop error

File: app/models/user.rb [hunk 1]
```diff
@@ -1,6 +1,22 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  has_many :reviews
+  has_many :cakes # !
```
Comment (OzgurFiratSayin) [inline app/models/user.rb]: I think dependent destroy should be here as the cakes we would like to be destroyed are those sold by the user

File: app/models/user.rb [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+  has_many :cakes, through: :orders, dependent: :destroy # not sure
```
Comment (OzgurFiratSayin) [inline app/models/user.rb]: Dependent destroy should be removed here I think, and one more association we need ,above this line, for this line to work is has_many :orders

**Review by OzgurFiratSayin — COMMENTED** [review]
Hi Natalia, I think we need both associations, so both lines should be in our script, I just left couple of line comments

**Pull request merged**